library/xml-model/src/test/java/org/cru/godtools/xml/model/tract/CardTest.kt

```diff
@@ -68,4 +74,11 @@ class CardTest {
         assertTrue(page.cards[1].isLastVisibleCard)
         assertFalse(page.cards[2].isLastVisibleCard)
     }
+
+    @Test
+    fun testCardBackgroundColorFallbackBehavior() {
+        val page = TractPage(Manifest(), cardBackgroundColor = Color.GREEN)
+        val card = Card(page, backgroundColor = null)
+        assertThat(card.backgroundColor, equalTo(page.cardBackgroundColor))
```

Comment:
This will test both fallback scenarios
```suggestion
        assertEquals(Color.GREEN, Card(page).backgroundColor)
        assertEquals(Color.BLUE, Card(page, backgroundColor = Color.BLUE).backgroundColor)
```